nifi-frontend/src/main/frontend/apps/nifi/src/app/pages/summary/ui/connection-status-listing/connection-status-table/connection-status-table.component.ts

```diff
@@ -124,12 +126,27 @@ export class ConnectionStatusTable extends ComponentStatusTable<ConnectionStatus
         return `${connection.connectionStatusSnapshot.percentUseCount}% | ${connection.connectionStatusSnapshot.percentUseBytes}%`;
     }
 
+    formatLoadBalanceStatus(connection: ConnectionStatusSnapshotEntity): string {
+        switch (connection.connectionStatusSnapshot.loadBalanceStatus) {
+            case 'LOAD_BALANCE_NOT_CONFIGURED':
+                return 'Not Configured';
+            case 'LOAD_BALANCE_ACTIVE':
+                return 'Active';
+            case 'LOAD_BALANCE_INACTIVE':
+                return 'Inactive';
+            default:
+                return 'unknown';
+
+            }
```

Comment:
To make this a bit more dynamic, I recommend simply removing `LOAD_BALANCE_` from the `loadBalanceStatus` string. That way `unknown` is not possible.

Reply:
Oops - I was logged in to GitHub as a different user. The @nifiguy comments came from @markobean.

Reply:
Good suggestion. I've updated to return `connection.connectionStatusSnapshot.loadBalanceStatus` in the default case.